ss13_tools/byond/key_tools.py

```diff
@@ -0,0 +1,54 @@
+from string import ascii_lowercase, digits
+from urllib.parse import quote
+from typing import Generator
+import asyncio
+import sys
+
+import requests as req
+from aiohttp import ClientSession
+
+from .constants import BYOND_MEMBERS_URL
+from ..constants import USER_AGENT
+
+
+def canonicalize(key: str) -> str:
+    """Turns a user's key into canonical form (ckey)"""
+    return ''.join([c for c in key.lower() if c in ascii_lowercase + digits + '@'])
```

Comment:
Is that a single-letter var I c? (probably should be char or letter)

Reply:
It is, I always use c instead of chat, since in most languages char is a reserved word. Will change it to letter.